Sources/Puree/Logger.swift

```diff
@@ -19,43 +19,42 @@ public final class Logger {
     }
 
     private let configuration: Configuration
+    private let dispatchQueue = DispatchQueue(label: "com.cookpad.Puree.Logger", qos: .background)
     private(set) var filters: [Filter] = []
     private(set) var outputs: [Output] = []
 
     public var currentDate: Date {
         return configuration.dateProvider.now
     }
 
-    var logStore: LogStore {
```

Comment:
Do you need to drop this property?

Reply:
Because this is not used and not in public scope. (this property is for testing but no tests use this.)

Reply:
I got